packages/circuit-ui/cli/migrate/__testfixtures__/typography-sizes.output.js

```diff
@@ -7,17 +7,18 @@ import {
   SubHeadline,
   Body,
   List,
+  Anchor,
 } from '@sumup/circuit-ui';
 
-const HeadlineZetta = () => <Headline size="zetta">Headline</Headline>;
```

Comment:
Was this left as `zetta` on purpose? What was the thinking behind it? If we leave it that way, at best the Headline will get the default size (too small for a legacy zetta heading) and at worst it will break TypeScript builds

Reply:
@amelako Do you remember the reason? I think it was because it should be migrated manually.

Reply:
Migrated to a custom size? But this could cause inconsistencies in the UI. Since `zetta` headings were the largest we had, they should probably become `one`, shouldn't they?

Reply:
I think so. Here's the [previous thread](https://github.com/sumup-oss/circuit-ui/pull/884#discussion_r619268345). Did you find the `zetta` size in the dashboard? I remember only seeing it in the marketing website where large headlines are dynamically sized anyway.

Reply:
As far as I remember we left it to migrate it manually to a custom size.

Reply:
Yeah I saw a couple of occurrences. So should I leave the codemod as-is and add a mention to the changelog to migrate manually? What should it be migrated to?

Reply:
Hm. It probably depends on the context, this might be good to review with a designer. If they say that most occurrences should be migrated to size `one`, then we could update the codemod.

Reply:
I talked with design, and we won't automate the `zetta` migration but will mention in the changelog/migration guide that it should be manually migrated either to a `one` if possible, or to a `rem` size for the time being.

Reply:
Done in 5be8a6a, replaced the transform with a warning for the Headline's `zetta` size and the Body's `giga` zize.